### ISSUE TITLE
[Comments] Quick Post - Building diagrams using graphviz

### ISSUE THREAD

**Issue by chadbaldwin** [chadbaldwin/chadbaldwin.github.io#13]

**Comment by kiquenet** [comment]
How-to Building dependency diagrams (Sql Server) using Graphviz ?
https://www.reddit.com/r/SQLServer/comments/mdt8gu/blog_building_dependency_diagrams_using_graphviz/

**Comment by chadbaldwin** [comment]
@kiquenet yup, that's my post on Reddit where I shared this blog post. Did you have any questions about it?

**Comment by PatrickGallucci** [comment]
Very nice Sir! I am going to use your example for a threat model tool I want to build. I will ping you with questions later.
